### PR TITLE
mini: add recipe

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **mini/***

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/mini/all/test_package/test_package.cpp [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+int main(void) {
+    mINI::INIFile file("test_package.ini");
+
+    mINI::INIStructure ini;
+
+    ini["things"]["chairs"] = "20";
+    ini["things"]["balloons"] = "100";
+
+    file.generate(ini);
```
Comment (RubenRBS) [inline recipes/mini/all/test_package/test_package.cpp:13]: Let's try to avoid writing the a file in the test_package if possible :)

**Review by RubenRBS — COMMENTED** [review]
Thanks! The recipe looks great, I only have a minor request in the test package :)

File: recipes/mini/all/test_package/test_package.cpp [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+}
```
Comment (jcar87) [inline recipes/mini/all/test_package/test_package.cpp:14]:
```suggestion
#include <iostream>

#include "mini/ini.h"

int main(void) {
    mINI::INIFile file("test_package.ini");

    std::cout << "mini test successful \n";
    return 0;
}
```

File: recipes/mini/all/test_package/test_package.cpp [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#include <iostream>
+
+#include "mini/ini.h"
+
+int main(void) {
+    mINI::INIFile file("test_package.ini");
```
Comment (jcar87) [inline recipes/mini/all/test_package/test_package.cpp:6]: @RubenRBS  I've updated the test_package - its a header only. library, so no need to check for linkage, and the constructor does not write a file to disk (it does nothing).

**Review by RubenRBS — APPROVED** [review]

**Pull request merged**